src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/ChangeSublayerVisibility/ChangeSublayerVisibility.xaml

```diff
@@ -1,30 +1,27 @@
 ﻿<UserControl x:Class="ArcGISRuntime.WPF.Samples.ChangeSublayerVisibility.ChangeSublayerVisibility"
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
-             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
-             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013" 
-             mc:Ignorable="d" 
-             d:DesignHeight="300" d:DesignWidth="300">
+             xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
     <Grid>
-        <esri:MapView x:Name="MyMapView"/>
+        <esri:MapView x:Name="MyMapView" />
         <Border
             Background="White" BorderBrush="Black" BorderThickness="1"
             HorizontalAlignment="Right" VerticalAlignment="Top"
             Margin="30" Padding="20" Width="375">
             <StackPanel>
-                <TextBlock Text="Change sublayer visibility" 
-                      Margin="0,0,0,2" TextWrapping="Wrap" />
+                <TextBlock Text="Select sublayers for display."
+                           TextWrapping="Wrap" TextAlignment="Center" FontWeight="SemiBold"
+                           Margin="0,0,0,5" />
                 <ListView x:Name="SublayerListView">
                     <ListView.ItemTemplate>
-                        <DataTemplate>
-                            <StackPanel Orientation="Horizontal">
-                                <CheckBox Content="{Binding Name}" IsChecked="{Binding IsVisible}" Margin="2"/>
-                            </StackPanel>
+                        <DataTemplate DataType="esri:ArcGISSublayer">
```

Comment:
I've always done `{x:Type esri:ArcGISSublayer}` but I see that this simpler syntax works just as well. Neat! TIL